pkg/crc/config/callbacks.go

```diff
@@ -32,3 +32,13 @@ func RequiresCRCSetup(key string, _ interface{}) string {
 	return fmt.Sprintf("Changes to configuration property '%s' are only applied during 'crc setup'.\n"+
 		"Please run 'crc setup' for this configuration to take effect.", key)
 }
+
+func RequiresHTTPPortChangeWarning(key string, value interface{}) string {
+	return fmt.Sprintf("Changes to configuration property '%s' will break the openshift route.\n"+
+		"User need to add %d port to application route to access it. like http://myapp.apps-crc.testing:%d", key, value, value)
+}
+
+func RequiresHTTPSPortChangeWarning(key string, value interface{}) string {
+	return fmt.Sprintf("Changes to configuration property '%s' will break the openshift webconsole.\n"+
```

Comment:
We need the same warning as in the HTTP case, plus something for the web console. `After this change, the OpenShift console will be non-functional because of OpenShift limitations` (dunno if it's possible to insert the URL of the web console in this message?)

Reply:
> We need the same warning as in the HTTP case, plus something for the web console. After this change, the OpenShift console will be non-functional because of OpenShift limitations

Sure, that we can do.

> dunno if it's possible to insert the URL of the web console in this message?

Not at this moment because web console url is created from bundle metadata side.